docs/reference/search/request/highlighting.asciidoc

```diff
@@ -27,7 +27,7 @@ GET /_search
     },
     "highlight" : {
         "fields" : {
-            "comment" : {}
+            "content" : {}
```

Comment:
is this change coming from another pr ?

Reply:
@jimczi  this is a typo noticed by someone else, I thought I would include this correction here as well. Let me know if  I should not do that.

Reply:
There is a pr open for this typo so we should use it rather than adding this in a pr that targets 6.x only.